test/e2e/antreapolicy_test.go

```diff
@@ -38,6 +38,7 @@ import (
 
```

Comment:
I think we can probably keep all the changes in `test/e2e/`

Reply:
Done.